tests/libxc/dft-anion/input.dat

```diff
@@ -0,0 +1,121 @@
+memory 1 Gb
+                                           
+molecule h2o {                             
+   -1 2                                    
+   O                                       
+   H 1 1.0                                 
+   H 1 1.0 2 104.5                         
+   units angstrom                          
+   no_reorient                             
+}                                          
+                                           
+set {                                      
+   reference uhf                           
+   basis 6-31G                             
+   dft_radial_points 100                   
+   dft_spherical_points 302                
+   guess sad                               
+   d_convergence 11                        
```

Comment:
`e_convergence` and `d_convergence` should both be set to 9 (a reasonable cutoff) radial and spherical should be bumped to 200/590 for testing.